gems/pending/util/miq-process.rb

```diff
@@ -127,24 +127,7 @@ def self.processInfo(pid = nil)
   end
 
   def self.command_line(pid)
-    case Sys::Platform::IMPL
-    when :mswin, :mingw
```

Comment:
Yeah, I had a striking suspicion it worked on Windows.  Oh, windows and the spaces...